### PR TITLE
 docs(deps): bump mkdocs-material to 7.1.6

### PR DESCRIPTION
**Pull request by wernerfred**
# Description

bump mkdocs-material to 7.1.6

Changelog:

Deprecated seealso admonition qualifier
Added Mongolian and updated Chinese translations
Fixed #2429: Version selector not touch-friendly on Android devices
Fixed #2703: Printed 'Initializing search' albeit ready on mobile

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (non-breaking change that does improve existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [x] If necessary I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by wernerfred** [comment]
I need to check if "Deprecated seealso admonition qualifier" has an impact on our docs first.

@polarathene we should enable preview builds also on changes to docs workflow files

**Comment by polarathene** [comment]
> I need to check if "Deprecated seealso admonition qualifier" has an impact on our docs first.

AFAIK it shouldn't. They're deprecating this in the docs, but I don't see any actual code in the project theme related to that.

The default admonition style is `note`, thus any qualifier that is not recognized should still fallback to `note`.

I don't really like the reasoning from the maintainer to deprecate it](https://github.com/squidfunk/mkdocs-material/commit/bcbe24224c9c8ad4f84edf70f4cfebd89c775fd3) ([related issue](https://github.com/squidfunk/mkdocs-material/issues/2700#issuecomment-848920694)), but my opinions often conflict with theirs. `seealso` still had useful semantic meaning, if it bothered anyone they could have just added a custom title or changed to a different qualifier :confused:

I suppose we should still replace any usage of `seealso` with `note`..

---

> @polarathene we should enable preview builds also on changes to docs workflow files

I didn't bother with that since for Pull Requests it'd require rebasing onto master IIRC?

At least for the `workflow_run` workflow that runs from `master` (thus no preview updates until merged and any existing PRs trigger a new run).

The `pull_request` workflow I think might work since I later learned that we had to implement the split workflow to resolve the security sandboxing Github applies to non-collaborator contributions. That should work with producing new builds for version bumps like this or updates to that workflow I guess? :+1:

**Review by polarathene — APPROVED** [review]

**Comment by wernerfred** [comment]
Added the sh script and the both preview files to the trigger section in https://github.com/docker-mailserver/docker-mailserver/commit/2083c615da2c423e4e9a04cece5560dbad888962 and it works i think. At least the bot commented with a preview link

File: .github/workflows/docs-preview-prepare.yml
Comment (polarathene) [inline .github/workflows/docs-preview-prepare.yml:9]: I don't think `docs-preview-deploy.yml` will matter. `workflow_run` should always be run from master version not anything contributed in a PR. So starting a workflow based on changes to that file is likely going to be misleading.

`docs-preview-prepare.yml` requires no secrets, and triggered the `workflow_run` event.

```suggestion
```

**Comment by wernerfred** [comment]
> I suppose we should still replace any usage of `seealso` with `note`..

Done within https://github.com/docker-mailserver/docker-mailserver/commit/3596ec20371fed7259f26364f0ba7c751c2a0ae7

Ready to merge i think, i will jsut take a quick look on the preiview again

**Review by polarathene — CHANGES_REQUESTED** [review]

**Review by polarathene — APPROVED** [review]

**Pull request merged**
